### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by btkelly**
- Cooldown bug fix
- Additional logging

### CODE REVIEW

File: amplify/src/main/java/com/github/stkent/amplify/tracking/checks/CooldownDaysCheck.java [hunk 1]
```diff
@@ -33,7 +33,8 @@ public CooldownDaysCheck(final long cooldownPeriodDays) {
 
     @Override
     public boolean shouldAllowFeedbackPrompt(@NonNull final Long cachedEventValue) {
-        return (SystemTimeUtil.currentTimeMillis() - cachedEventValue) >= TimeUnit.DAYS.toMillis(cooldownPeriodDays);
+        return cachedEventValue == Long.MAX_VALUE
+                || (SystemTimeUtil.currentTimeMillis() - cachedEventValue) >= TimeUnit.DAYS.toMillis(cooldownPeriodDays);
```
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/checks/CooldownDaysCheck.java]: Does this mean we need to change the warmup days check? does that even exist still?

File: amplify/src/main/java/com/github/stkent/amplify/tracking/managers/ApplicationChecksManager.java [hunk 2]
```diff
@@ -74,16 +74,36 @@ public boolean shouldAllowFeedbackPrompt() {
...
+            boolean installResult = installTimeEventCheck.shouldAllowFeedbackPrompt(installTime);
+
+            if (!installResult) {
+                logger.d("Blocking prompt based on install time check");
+            }
+
+            result = result && installResult;
         }
 
         if (lastUpdateTimeTimeEventCheck != null) {
```
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/managers/ApplicationChecksManager.java]: Be a darling and fix that name for me please!

File: amplify/src/main/java/com/github/stkent/amplify/tracking/checks/CooldownDaysCheck.java (hunk 1 above)
Comment (btkelly) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/checks/CooldownDaysCheck.java]: I don't think so, I looked at that and it seemed correct there

File: amplify/src/main/java/com/github/stkent/amplify/tracking/managers/ApplicationChecksManager.java (hunk 2 above)
Comment (btkelly) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/managers/ApplicationChecksManager.java]: lol, well you have to give me the new name if you want it changed

Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/managers/ApplicationChecksManager.java]: lastUpdateTimeTimeEventCheck -> lastUpdateTimeEventCheck

burn-it

**Pull request merged**